client/packages/common/src/ui/layout/tables/components/Cells/NumberInputCell/NumberCell.tsx

```diff
@@ -11,10 +11,18 @@ export const NumberCell = <T extends RecordWithId>({
   column,
   rowData,
   isError,
-}: CellProps<T>) => {
+  defaultValue,
```

Comment:
I wonder if we should set a default `defaultValue` of empty string? Currently it displays `0` if a default isn't defined, which seems slightly misleading.